pkg/util/tracing/tracingui/html_template.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <style>
+    .sort.asc, .sort.desc {
+      background-color: yellow;
+    }
+    .sort.asc::after {
+      content: "\002B06";
+      padding-left: 3px;
+    }
+    .sort.desc::after {
+      content: "\002B07";
+      padding-left: 3px
+    }
+
+    .link-button {
+      background: none!important;
+      border: none;
+      padding: 0!important;
+      font-family: arial, sans-serif;
+      color: #069;
+      text-decoration: underline;
+      cursor: pointer;
+    }
+
+  </style>
+</head>
+
+<script src="list.js"></script>
```

Comment:
Would it be okay to request this library from CDN like this:
```<script src="//cdnjs.cloudflare.com/ajax/libs/list.js/2.3.1/list.min.js"></script>```
It won't be necessary to embed the JS file and treat it as a local resource.